source/guides/clear/swupd-3rd-party.rst

```diff
@@ -0,0 +1,304 @@
+.. _swupd-3rd-party:
+
+swupd 3rd-party 
+###############
+
+Upstream |CL| offers a plethora of `bundles`_ to choose from.  
+
+To provide you access to even more software outside of the distro, 
+the :ref:`mixer` and :ref:`swupd-guide` tools have added support for 3rd-party
+bundles starting with release 32570.  
+
+:command:`mixer` can now create 3rd-party bundles out of any content you want 
+without the need to package it into an RPM format first.
+
+And to consume and manage these bundles, a :command:`3rd-party` 
+subcommand has been added to :command:`swupd`.
+
+Follow this guide to see how to build an example 3rd-party bundle, set up a web 
```

Comment:
Ln 18-19 is a duplicate of Ln 21-23.  Will you please decide which takes precedence?